internal/volumes/mount.go

```diff
@@ -134,7 +138,16 @@ func (s *LinuxMountService) Publish(targetPath string, devicePath string, opts M
 		return s.mounter.Mount(devicePath, targetPath, opts.FSType, mountOptions)
 	}
 
-	return s.mounter.FormatAndMount(devicePath, targetPath, opts.FSType, mountOptions)
+	formatOptions := make([]string, 0)
+
+	if opts.FormatOptions != "" {
+		formatOptions = strings.Split(opts.FormatOptions, " ")
```

Comment:
This is not safe, please use a shell lexer to split the arguments, or don't split them at all.

Reply:
Or use a list of argument instead of a string as parameter.